packages/design-system/src/interfaces/forms.d.ts

```diff
@@ -181,6 +181,13 @@ export interface ISearchBarProps extends ISearchBarHtmlProps {
   onExpand?(): void;
 }
 
+export interface IIconProps {
+  icon: React.ReactNode;
```

Comment:
that is not the right type if you want to handle require()

Reply:
Can you tell me what's the proper type for handling `required` as I can't seem to find it :(